### PR TITLE
[release] merging crate-release-update into crate

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @GuillaumeGomez @EPashkin @sdroege

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -21,27 +21,25 @@ travis-ci = { repository = "gtk-rs/glib" }
 name = "glib"
 
 [dependencies]
-lazy_static = "1.0"
+lazy_static = "1.2"
```
Comment (sdroege) [inline Cargo.toml]: Why? Do we fail to compile with `1.0` now?

Comment (sdroege) [reply Cargo.toml]: Part of 68a03649d6ff9f91ef46f879d30d84bd374b36e8 from master. Has no justification but it's in master so will be fine ;)

CC @EPashkin

Comment (GuillaumeGomez) [reply Cargo.toml]: No clue. It's just a mirror of what we have on master after all.

**Comment by sdroege** [comment]
:+1:

**Pull request merged**